### ISSUE TITLE
Storage 

### ISSUE THREAD

**Issue by Ty0ne** [Minehut/Meta#898]
### Feature request

**Checklist**
- [x] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [x] This is not a bug report, support request or plugin request
- [x] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [x] This issue does not contain any sensitive information 

**Platform**
Website

**Describe the feature**
Somewhere on the Server Dashboard preferably on the top, there needs to be a piece of text that shows you how much storage your server has remaining out of the 20GB limit given for your Server additionally if your server runs out of storage then it should not basically brick it instead if you start your server it should come out of Hibernation but remain offline so you can free up some storage on your server and not have to contact Minehut Support to delete files for you when this happens.

**How the feature is useful**
Let's say you operate a Survival Server that has players load chunks there is no good way to check how much storage your server has remaining besides manually downloading the server and checking which is pretty annoying todo and if your server reaches the 20GB limit then you need some downtime depending on Minehut Support's response time.

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by Checkmate128** [comment]
Yes, having access to basic server metrics such as storage is important for server owners.

**Comment by Prorickey** [comment]
Duplicate of #622
